models/api.models.js

```diff
@@ -47,9 +47,23 @@ exports.fetchCommentsByArticleId = (id) => {
 	}
 	let sqlQuery = `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`;
 	return db.query(sqlQuery, [id]).then((result) => {
-		if (result.rows.length === 0) {
+		return result.rows;
+	});
+};
+
+exports.postCommentByArticleId = (username, body, id) => {
+	if (isNaN(id)) {
+		return Promise.reject({ status: 400, msg: 'Bad Request' });
+	}
+	const queryParams = [body, username, id];
+	const checkUsers = `SELECT * FROM users WHERE username = $1`;
+	return db.query(checkUsers, [username]).then(({ rows }) => {
```

Comment:
If the user does not exist you will get a psql error (foreign key violation - so again you can handle this differently)